Sources/FileSystem/Model/BetaTester.swift

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import CodableCSV
+import Foundation
+import Model
+
+public struct BetaTester: Codable, Equatable, Hashable {
+    public let email: String
+    public let firstName: String
+    public let lastName: String
+    public let inviteType: String
```

Comment:
Wondering if we really care about the invite type.  What would the consequence be of removing that?

Reply:
It doesn't matter, I think, `inviteType` is just a property to show it was invited by `Email` or `PublicLink`. Do we want to remove it?

Reply:
I think we should remove it given that all users we add to the configuration will be invited by email.  And if the user already exists they have have been invited.